Momento/Incubating/SimpleCacheClientFactory.cs

```diff
@@ -0,0 +1,10 @@
+namespace MomentoSdk.Incubating;
+
+public class SimpleCacheClientFactory
+{
+    public static SimpleCacheClient Get(string authToken, uint defaultTtlSeconds, uint? dataClientOperationTimeoutMilliseconds = null)
```

Comment:
Ah, here's the builder :)  ignore previous comment.

Is this a typical naming convention (`*Factory.Get`) for builders in CS?

Reply:
> Is this a typical naming convention (*Factory.Get) for builders in CS?

I poked around and didn't see a consensus. I was learning toward `CreateClient` as opposed to `Get`. Thoughts?